src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,185 @@
+  </figcaption>
+</figure>
+
+The `backdrop-filter` property is like CSS [filters](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) in that all your favorite [filter functions](https://developer.mozilla.org/en-US/docs/Web/CSS/filter#Filter_functions) are supported: `blur()`, `brightness()`, `contrast()`, `opacity()`, `drop-shadow()`, and so on. It also supports the `url()` function if you want to use an external image as the filter, as well as the keywords `none`, `inherit`, `initial`, and `unset`. There are explanations for all of this on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter), including descriptions of syntax, filters, and values.
+
+When `backdrop-filter` is set to anything other than `none`, the browser creates a new [stacking context](https://www.w3.org/TR/CSS21/zindex.html). A [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) may also be created, but only if the element has absolute and fixed position descendants.
+
+You can combine filters for rich and clever effects, or use just one filter for more subtle or precise effects. You can even combine them with [SVG filters](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/filter).
+
+As with many features of the modern web, you'll want to know whether the user's browser supports `backdrop-filter` before using it. Do this with `@supports()`:
+
+```css
+@supports (-webkit-backdrop-filter: none) or (backdrop-filter: none) {
+	.ManipulateElement {
+		backdrop-filter: blur(10px);
+		-webkit-backdrop-filter: blur(10px);
+	}
+}
+```
+
+## Examples
+
+Design techniques and styles previously reserved for native operating systems are now performant and achievable with a single CSS declaration. Let's look at some examples.
+
+### Single filter
+
+In the following example, the frosted effect is achieved by combining color and blur. The blur is supplied by `backdrop-filter`, while the color comes from the element's semi-transparent background color.
+
+```css
+.blur-behind-me {
+  background-color: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(.5rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Try this example for yourself in <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>.
+  </figcaption>
+</figure>
+
+### Multiple filters
+
+Sometimes you'll need multiple filters to achieve the desired effect. To do this, provide a list of filters separated by a space. For example:
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+In the following example, each of the four panes have different combinations of backdrop filters while the same set of shapes are animated behind them.
```

Comment:
```suggestion
In the following example, each of the four panes has a different combination of backdrop filters while the same set of shapes are animated behind them.
```
Number agreement error: _each_ is the subject of the verb _has_ since _of the four panes_ is a prepositional phrase.